src/panels/lovelace/hui-root.js

```diff
@@ -76,7 +84,7 @@ class HUIRoot extends NavigateMixin(EventsMixin(PolymerElement)) {
         </div>
       </app-header>
 
-      <span id='view'></span>
+      <span id='view' class$="[[_getViewClass(_tabsHidden)]]"></span>
```

Comment:
maybe we can replace `_getViewClass()` and `_computeTabsHidden()` with `.classList.toggle()` in `_configChanged`

Reply:
Should we do it in `_computeTabsHidden()` instead since that is watching `views`?